core/src/main/java/org/apache/carbondata/core/view/MVManager.java

```diff
@@ -143,9 +143,9 @@ public void createSchema(String databaseName, MVSchema viewSchema)
   /**
    * Drops the mv schema from storage
    *
-   * @param viewName index name
+   * @param viewName mv name
    */
-  public void deleteSchema(String databaseName, String viewName) throws IOException {
+  public synchronized void deleteSchema(String databaseName, String viewName) throws IOException {
```

Comment:
why we need `synchronized` here? As i can see, if two people are dropping same, then one will be successful and other will get exception as MV does not exists. Any special case or scenario you observed?

Reply:
added so that another thread could skip some steps. as this occurs only few times when multiple threads run, removing it.